src/functions-and-arrays.js

```diff
@@ -121,15 +121,30 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arr) {} 
```

Comment:
The logic is pretty much what you've done here, but you don't need to use `.concat()` neither `.sort()` you just need to `.push()` the items into the new `array` if that `word doesn't exist` in the array:

```
function uniquifyArray(arr) {
  if (arr.length === 0) {
    return null;
  }
  let uniqueArr = [];
  for (let i = 0; i < arr.length; i++) {
    if (!uniqueArr.includes(arr[i])) {
      uniqueArr.push(arr[i]);
    }
  }
  return uniqueArr;
}
```